eclair-core/src/test/resources/integration/bitcoin.conf

```diff
@@ -1,11 +1,13 @@
 regtest=1
+noprinttoconsole=1 
 server=1
 port=28333
```

Comment:
`port` is also a section setting: https://bitcoincore.org/en/releases/0.17.0/#configuration-sections-for-testnet-and-regtest

Though, given that the tests pass, setting the port is probably not needed at all